### PR TITLE
Replace y2base

### PR DESCRIPTION
**Pull request by jreidinger**
and as bonus drop some unused tools

### CODE REVIEW

File: base/src/y2base.cc [hunk 1]
```diff
@@ -1,22 +0,0 @@
...
-
-   File:       y2base.cc
-
-   Author:     Mathias Kettner <kettner@suse.de>
-   Maintainer: Mathias Kettner <kettner@suse.de>
-
-/-*/
-/*
- * Hi. This file is empty on purpose :-)
- * Nice, isn't it?
```
Comment (mvidner) [inline base/src/y2base.cc]: Well, that means we are not really dropping the old code. The old code is in [liby2/src/genericfrontend.cc](https://github.com/yast/yast-core/blob/a0fefb49b7497a21624fd6f65a35de56b2dbe74a/liby2/src/genericfrontend.cc). Why don't we drop that too?

Comment (jreidinger) [inline base/src/y2base.cc]: well, in past it was used also for scr binary. But we probably also no longer use it. So it maybe also make sense to drop it.

File: package/yast2-core.changes [hunk 2]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Feb 28 15:16:58 UTC 2017 - jreidinger@suse.com
+
+- Drop y2base binary which is replaced by y2base ruby script from
+  yast2-ruby-bindings (bsc#1027181)
```
Comment (mvidner) [inline package/yast2-core.changes]: Also drop the no longer used md_autorun and tty_wrapper.

**Comment by jreidinger** [comment]
in the end this pr is closed as y2base is still needed for yast2-testsuite. So we will drop it when old testsuite will be dropped.

**Pull request closed without merging**